src/test/java/org/kiwiproject/dropwizard/util/health/MonitoredJobHealthCheckTest.java

```diff
@@ -0,0 +1,270 @@
+package org.kiwiproject.dropwizard.util.health;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.kiwiproject.test.assertj.dropwizard.metrics.HealthCheckResultAssertions.assertThatHealthCheck;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.dropwizard.util.Duration;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.kiwiproject.base.KiwiEnvironment;
+import org.kiwiproject.dropwizard.util.job.MonitoredJob;
+
+import java.time.Instant;
+import java.util.concurrent.atomic.AtomicLong;
+
+@DisplayName("MonitoredJobHealthCheck")
+class MonitoredJobHealthCheckTest {
+
+    private MonitoredJob job;
+    private long now;
+
+    @BeforeEach
+    void setUp() {
+        job = mock(MonitoredJob.class);
+        now = System.currentTimeMillis();
+    }
+
+    @Nested
+    class Builder {
+
+        @Test
+        void shouldRequireMonitoredJob() {
+            var builder = MonitoredJobHealthCheck.builder();
+            assertThatThrownBy(builder::build)
```

Comment:
Not big deal but for this and below you could use `assertThatIllegalArgumentException`